doc/source/dev/roadmap.rst

```diff
@@ -50,13 +50,16 @@ requested in this respect.
 Statistics enhancements
 -----------------------
 
-The `scipy.stats` enhancements listed in the :ref:`scipy-roadmap-detailed` are of
-particularly high importance to the project.
-
-- Improve the options for fitting a probability distribution to data.
-- Expand the set of hypothesis tests.  In particular, include all the basic
-  variations of analysis of variance.
-- Add confidence intervals for all statistical tests.
+The following `scipy.stats` enhancements and those listed in the
+:ref:`scipy-roadmap-detailed` are of particularly high importance to the
+project.
+
+- Overhaul the univariate distribution infrastructure: each distribution
+  family will be represented by a *class* rather than a distribution *object*,
+  and specific distributions will be represented by instantiating the class
+  rather than "freezing" the distribution via the ``__call__`` method.
```

Comment:
This would make it more practical to address most of the issues listed at gh-15928. See some additional discussion [here](https://github.com/tirthasheshpatel/scipy/pull/14#issuecomment-1100914807).

Reply:
We will get huge push backs here, so it's good that you have something prepared. +100 for me on this one.